modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -167,6 +167,10 @@ PolygonConcentricCircleMeshGeneratorBase::validParams()
                         "Whether the side elements are reorganized to have a uniform size.");
   params.addParam<bool>(
       "quad_center_elements", false, "Whether the center elements are quad or triangular.");
+  params.addRangeCheckedParam<Real>(
+      "center_quad_factor",
+      "center_quad_factor>0&center_quad_factor<1",
+      "A factor used to support customized quad center nodes determination.");
```

Comment:
needs to be more descriptive.
It's a ratio of distances right

Reply:
let's adjust the 3 docstrings with the more descriptive too please

Reply:
Yes, may be call it "a factor used to determine the locations of transition nodes in the center region meshed by quad elements"?

Reply:
it still doesnt explicitly say how it determines this. 

btw would it fit in one of the parameter groups?

Reply:
seems good to me.

So the range check bans 0 so this part is confusing. Users will try to set a 0 to get the default behavior. 
I would rather we dont talk about this 0, just say: default value is ....